app/controllers/collections_controller.rb

```diff
@@ -137,6 +139,8 @@ def web_create_collection
     @collection_title = params[:collection_title]
     @collection_owner = params[:collection_owner]
     @collection_abstract = params[:collection_abstract]
+    @approval_required = params[:approval_required]
+    @approval_required = 'private' if request.get?
```

Comment:
The GET part is for initialising the form input with default values. In this case it is setting the privacy checkbox to be true by default (whereas all the text fields are empty by default). The POST parts are for the form submission. The stuff outside the POST and GET is for re-populating the form with any previous input in the event where the form submission fails and the user has  to correct some invalid input.